ops/stripe_plans.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+var async = require("async");
+var currency_data = require("../data/currencies");
+var currencies = Object.keys(currency_data);
```

Comment:
Maybe make the top two lines one and remove the currency_data var?

Is there something significant to the name "currency_data" or could it be "currenctData"?